docs/guide/troubleshooting.md

```diff
@@ -51,6 +51,24 @@ If the above steps don't work, you can try adding `DefaultLimitNOFILE=65536` as
 
 Note that these settings persist but a **restart is required**.
 
+### Randomly stops working
```

Comment:
"randomly stops working" is a bit vague. in what way does it stop working? do requests hang, are there exceptions, does the process crash, etc.?